src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController.swift

```diff
@@ -14,58 +15,98 @@
 // under the License.
+//
 
-import Foundation
 import UIKit
 
 class AppInformationViewController: UITableViewController {
-	
-	@IBOutlet weak var labelVersion: UILabel!
-	
-	override func viewDidLoad() {
-		super.viewDidLoad()
-		updateVersionLabel()
-	}
-	
-	override func prepare(for segue: UIStoryboardSegue, sender _: Any?) {
-		let destination = segue.destination
-
-		guard
-			let segueIdentifier = segue.identifier,
-			let segue = SegueIdentifier(rawValue: segueIdentifier)
-		else { return }
-
-		switch segue {
-		case .about:
-			(destination as? AppInformationDetailViewController)?.model = .about
-		case .contact:
-			(destination as? AppInformationDetailViewController)?.model = .contact
-		case .help:
-			(destination as? AppInformationHelpViewController)?.model = .questions
-		case .legal:
-			(destination as? AppInformationDetailViewController)?.model = .legal
-		case .privacy:
-			(destination as? AppInformationDetailViewController)?.model = .privacy
-		case .terms:
-			(destination as? AppInformationDetailViewController)?.model = .terms
-		}
-	}
-	
-	func updateVersionLabel() {
-		guard let bundleVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] else { return }
-		guard let bundleBuild = Bundle.main.infoDictionary?["CFBundleVersion"] else { return }
-		
-		self.labelVersion.text = "\(AppStrings.Home.appInformationVersion) \(bundleVersion) (\(bundleBuild))"
+
+	// MARK: - Properties
+	private let model: [AppInformationDetailModel] = [
+		.about,
+		.faq,
+		.terms,
+		.privacy,
+		.contact,
+		.legal
+	]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+		navigationItem.largeTitleDisplayMode = .always
+		title = "Home_AppInformationCard_Title".localized
+
+        setupTableView()
+    }
+
+	private func setupTableView() {
+		tableView.contentInset = .init(top: 30, left: 0, bottom: 0, right: 0)
+		tableView.backgroundColor = .preferredColor(for: .backgroundSecondary)
+
+		tableView.register(UITableViewCell.self, forCellReuseIdentifier: "appInformationCell")
+
+		tableView.sectionFooterHeight = UITableView.automaticDimension
+		tableView.estimatedSectionFooterHeight = 20
+		tableView.tableFooterView = UIView()
 	}
+
 }
 
 extension AppInformationViewController {
-	private enum SegueIdentifier: String {
-		case about = "aboutSegue"
-		case contact = "contactSegue"
-		case legal = "legalSegue"
-		case privacy = "privacySegue"
-		case terms = "termsSegue"
-		case help = "helpSegue"
+
+	override func numberOfSections(in _: UITableView) -> Int {
+		1
+	}
+
+	override func tableView(_: UITableView, titleForHeaderInSection section: Int) -> String? {
+		""
+	}
+
+	override func tableView(_: UITableView, numberOfRowsInSection section: Int) -> Int {
+		model.count
+	}
+
+	override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		let cell = tableView.dequeueReusableCell(withIdentifier: "appInformationCell", for: indexPath)
```

Comment:
Would you mind using `TableViewCellReuseIdenfitiers` here? Would avoid typos in case we have to touch this again.